Console/MWD.cpp

```diff
@@ -0,0 +1,678 @@
+                                                                                                                KeyBoard(' ',1);
+        }else{												KeyBoard(' ',0);
+                                                                                                                KeyBoard(0x10,0);
+        }
+        if(maskData[5] & 0x80) if(lastStateData[1] & 0x80) 	{
+                                                                                                                KeyBoard(0x10,1);
+                                                                                                                KeyBoard('F',1);
+        }else{												KeyBoard('F',0);
+                                                                                                                KeyBoard(0x10,0);
+        }//*/
+
+    /* NASTAWNIK, BOCZNIK i KIERUNEK */
+
+    if(bnkMask & 1){    //puszczanie klawiszy
+        KeyBoard(0x6B,0);
+        bnkMask &= ~1;
+    }
+    if(bnkMask & 2){
+        KeyBoard(0x6D,0);
+        bnkMask &= ~2;
+    }
+    if(bnkMask & 4){
+        KeyBoard(0x6F,0);
+        bnkMask &= ~4;
+    }
+    if(bnkMask & 8){
+        KeyBoard(0x6A,0);
+        bnkMask &= ~8;
+    }
+
+    if(nastawnik < ReadDataBuff[6])
+    {
+        bnkMask |= 1;
+        nastawnik++;
+        KeyBoard(0x6B,1); //wci�nij + i dodaj 1 do nastawnika
+    }
+    if(nastawnik > ReadDataBuff[6])
+    {
+        bnkMask |= 2;
+        nastawnik--;
+        KeyBoard(0x6D,1); //wci�nij - i odejmij 1 do nastawnika
+    }
+    if(bocznik < ReadDataBuff[7])
+    {
+        bnkMask |= 4;
+        bocznik++;
+        KeyBoard(0x6F,1); //wci�nij / i dodaj 1 do bocznika
+    }
+    if(bocznik > ReadDataBuff[7])
+    {
+        bnkMask |= 8;
+        bocznik--;
+        KeyBoard(0x6A,1); //wci�nij * i odejmij 1 do bocznika
+    }
+
+    /* Obs�uga HASLERA */
+    if(ReadDataBuff[0] & 0x80) bCzuwak = true;
+
+    if(bKabina1){			// logika rysika 1
+        bRysik1H = true;
+        bRysik1L = false;
+        if(bPrzejazdSHP) bRysik1H = false;
+    }else if(bKabina2){
+        bRysik1L = true;
+        bRysik1H = false;
+        if(bPrzejazdSHP) bRysik1L = false;
+    }else{
+        bRysik1H = false;
+        bRysik1L = false;
+    }
+
+    if(bHamowanie){			// logika rysika 2
+        bRysik2H = false;
+        bRysik2L = true;
+    }else{
+        if(bCzuwak) bRysik2H = true;
+        else bRysik2H = false;
+        bRysik2L = false;
+    }
+    bCzuwak = false;
+    if(bRysik1H) 	WriteDataBuff[6] |= 1<<0;
+    else 			WriteDataBuff[6] &= ~(1<<0);
+    if(bRysik1L) 	WriteDataBuff[6] |= 1<<1;
+    else 			WriteDataBuff[6] &= ~(1<<1);
+    if(bRysik2H) 	WriteDataBuff[6] |= 1<<2;
+    else 			WriteDataBuff[6] &= ~(1<<2);
+    if(bRysik2L) 	WriteDataBuff[6] |= 1<<3;
+    else 			WriteDataBuff[6] &= ~(1<<3);
+}
+
+bool MWDComm::KeyBoard(int key, int s)	// emulacja klawiatury

```

Comment:
W jakim celu funkcja zwraca bool jeśli zawsze jest to true?
Może zamiast int s dać bezpośrednie użycie keyeventa?